lms/djangoapps/django_comment_client/base/views.py

```diff
@@ -63,6 +66,36 @@ def ajax_content_response(request, course_key, content):
     })
 
 
+def track_forum_event(request, event_name, course, obj, data):
+    """
+    Send out an analytics event when a forum event happens. Works for threads,
+    responses to threads, and comments on those responses.
+    """
+    user = request.user
+    data['id'] = obj.id
+    id_map = get_discussion_id_map(course)
```

Comment:
this happens to be a costly call to make, is it absolutely necessary?  i guess this is so the name can be enclosed along with the commentable id, for convenience?

also, as much as i hate the term 'commentable' we already have a problem with multiple names for this thing ('discussion', 'topic' etc) - is 'category' being deliberately standardized upon for analytics?